clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/description/DeployCloudFoundryServiceDescription.java

```diff
@@ -28,4 +29,5 @@
   String servicePlan;
   Set<String> tags;
   String parameters;
+  Map<String, Object> parameterMap;
```

Comment:
Why is there both `parameters` and `parameterMap`?

Reply:
We removed the parameters field and we convert to the parameterMap in the converter.